### PR TITLE
More informative error message when a install_ function cannot be found.

### PR DESCRIPTION
**Pull request by jimhester**
Previously this gave the same error as other Malformed remotes, this should point users to the proper problem.

### CODE REVIEW

File: R/deps.R [hunk 1]
```diff
@@ -167,13 +167,14 @@ dev_remote_type <- function(remotes = "") {
...
+    fun_nme <- paste0("install_", tolower(type))
     tryCatch(
-      fun <- get(x = paste0("install_", tolower(type)),
+      fun <- get(x = fun_nme,
         envir = asNamespace("devtools"),
         mode = "function",
         inherits = FALSE),
       error = function(e) {
-        stop("Malformed remote specification '", x, "'", call. = FALSE)
+        stop("Malformed remote specification '", x, "', 'devtools:::", fun_nme, "' does not exist", call. = FALSE)
```
Comment (hadley) [inline R/deps.R]: How about just `stop("Unknown remote type")`?

Comment (jimhester) [inline R/deps.R]: Works for me, done in https://github.com/jimhester/devtools/commit/7b378ee0e6a2afcbc21172e61a790f613f335b15

File: R/deps.R [hunk 2]
```diff
@@ -167,13 +167,14 @@ dev_remote_type <- function(remotes = "") {
...
+    fun_nme <- paste0("install_", tolower(type))
     tryCatch(
-      fun <- get(x = paste0("install_", tolower(type)),
+      fun <- get(x = fun_nme,
         envir = asNamespace("devtools"),
         mode = "function",
         inherits = FALSE),
       error = function(e) {
-        stop("Malformed remote specification '", x, "'", call. = FALSE)
+        stop("Unknown remote type", call. = FALSE)
```
Comment (hadley) [inline R/deps.R]: And maybe put the type into just to be helpful?

Comment (jimhester) [inline R/deps.R]: Done! https://github.com/hadley/devtools/commit/f4151b9da8e68051eadc4e9cc452648c520585e6

**Comment by hadley** [comment]
Thanks!

**Pull request merged**